src/modules/location/data/france/FranceCities.h

```diff
@@ -1004,5 +1004,5 @@ const std::vector<std::string> franceCities = {"Abbeville",
                                                "Yerres",
                                                "Yutz",
                                                "Yvetot",
-                                               "Yzeure"};  
-}
```

Comment:
@cieslarmichal Are there any commands that need to run to reformat the code before committing?

Reply:
clang-format src/**/*.cpp src/**/*.h include/**/*.h -i -style=file

I will add it to readme